package/com.unity.formats.usd/Editor/Scripts/Behaviors/UsdMenu.cs

```diff
@@ -148,267 +176,365 @@ public class UsdMenu : MonoBehaviour {
-        importOptions.materialImportMode = MaterialImportMode.ImportDisplayColor;
-        importOptions.usdRootPath = GetDefaultRoot(scene);
-      }
-
-      GameObject root = new GameObject(GetObjectName(importOptions.usdRootPath, path));
-
-      if (Selection.gameObjects.Length > 0) {
-        root.transform.SetParent(Selection.gameObjects[0].transform);
-      }
-
-      try
-      {
-        UsdToGameObject(root, scene, importOptions);
-        return root;
-      }
-      catch (SceneImporter.ImportException)
-      {
-        GameObject.DestroyImmediate(root);
-        return null;
-      }
-    }
-
-    [MenuItem("USD/Import as Prefab", priority = 1)]
-    public static void MenuImportAsPrefab() {
-      var scene = InitForOpen();
-      if (scene == null) {
-        return;
-      }
-      string path = scene.FilePath;
-
-      // Time-varying data is not supported and often scenes are written without "Default" time
-      // values, which makes setting an arbitrary time safer (because if only default was authored
-      // the time will be ignored and values will resolve to default time automatically).
-      scene.Time = 1.0;
-
-      var importOptions = new SceneImportOptions();
-      importOptions.projectAssetPath = GetSelectedAssetPath();
-      importOptions.changeHandedness = BasisTransformation.SlowAndSafe;
-      importOptions.materialImportMode = MaterialImportMode.ImportDisplayColor;
-      importOptions.usdRootPath = GetDefaultRoot(scene);
 
-      var invalidChars = Path.GetInvalidFileNameChars();
-      var prefabName = string.Join("_", GetPrefabName(path).Split(invalidChars,
-          System.StringSplitOptions.RemoveEmptyEntries)).TrimEnd('.');
-      string prefabPath = importOptions.projectAssetPath + prefabName + ".prefab";
-      prefabPath = AssetDatabase.GenerateUniqueAssetPath(prefabPath);
-      string clipName = Path.GetFileNameWithoutExtension(path);
 
-      var go = new GameObject(GetObjectName(importOptions.usdRootPath, path));
-      try {
-        UsdToGameObject(go, scene, importOptions);
-        SceneImporter.SavePrefab(go, prefabPath, clipName, importOptions);
-      } finally {
-        GameObject.DestroyImmediate(go);
-        scene.Close();
-      }
-    }
-
-    [MenuItem("USD/Import as Timeline Clip", priority = 2)]
-    public static void MenuImportAsTimelineClip() {
-      var scene = InitForOpen();
-      if (scene == null) {
-        return;
-      }
-
-      string path = scene.FilePath;
-
-      var invalidChars = Path.GetInvalidFileNameChars();
-      var prefabName = string.Join("_", GetPrefabName(path).Split(invalidChars,
-          System.StringSplitOptions.RemoveEmptyEntries)).TrimEnd('.');
+        [MenuItem(UsdPrefPayloadMenuName, priority = 0)]
+        static void LoadAllPayloads()
+        {
+            var enabled = Menu.GetChecked(UsdPrefPayloadMenuName);
+            EditorPrefs.SetBool(PayloadPolicyPrefName, !enabled);
```

Comment:
I suggest you "steal" my SavedBool class altogether: that allows you to create it with your funky ID, and from that point onwards you treat as a regular bool. also that class can store the bool data. the menu just adds the tick according to this value.

Reply:
Added in #9b02130